packages/api-core/src/resources/settings.js

```
@@ -15,5 +17,41 @@ export default class AvSettings extends AvApi {
       merge,
       config: options,
     });
+    this.avUsers = avUsers;
```

Probably should throw is `avUser` is not provided here.

Or throw in the methods where it is used so that the other methods can still be used without needing it (maybe a less breaking change).

---

I added it to the other implementations, that pattern makes sense but might be worth saving for a refactor PR for all the other apis with similar dependencies.

---

@KaseyPowers changes LGTM. Can you open a ticket with the refactor suggestions?

---

Created #66